storybook/package.json

```diff
@@ -10,15 +10,17 @@
 	"scripts": {
 		"prebuild": "npm run remove-dist",
 		"build": "storybook build -c . -o ../dist --quiet --loglevel warn",
-		"start": "storybook dev -p 9001 -c . --no-version-updates",
-		"remove-dist": "rimraf ../dist/"
+		"start": "storybook dev -p 6006 -c . --no-version-updates",
+		"remove-dist": "rimraf ../dist/",
+		"test-storybook": "test-storybook --config-dir ."
 	},
 	"devDependencies": {
 		"@babel/preset-react": "^7.26.3",
 		"@storybook/addon-viewport": "^8.6.12",
 		"@storybook/addon-webpack5-compiler-babel": "^3.0.6",
 		"@storybook/react-webpack5": "^8.6.12",
 		"@storybook/test": "^8.6.12",
+		"@storybook/test-runner": "^0.19.1",
```

Comment:
The current version is 0.23.0. I'm guessing it's been updated since this was authored, but I don't think the `^` will update to the latest when the version is less than 1.0.